modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -95,7 +95,7 @@ public void processOpts() {
 
         // default this to true so the python ModelSimple models will be generated
         ModelUtils.setGenerateAliasAsModel(true);
-        LOGGER.info(CodegenConstants.GENERATE_ALIAS_AS_MODEL + " is hard coded to true in this generator. Alias models will only be generated if they contain validations or enums");
+        LOGGER.info(CodegenConstants.GENERATE_ALIAS_AS_MODEL + " is hard coded to true in this generator.");
```

Comment:
Now that the Java generator connects all request.body parameters to the correct aliasModels, we need to connect up all of those alias models in:
- request.body
- response.body
Earlier, python-experimental was deleting alias models that did't have validations or enums.
Now that Java is using all alias models, even those without validations and enums, this generator should use all of those alias models.